docs/ci/demo

```diff
@@ -38,6 +38,9 @@ popper run --timeout "1m 10s"
 # arbitrary stages and envs
 init_test
 popper init --stages=one,two,three --envs=host,ubuntu-xenial pipetwo
+popper init --stages=one,two,teardown sample_exp
+popper init --stages=one,teardown,three sample_exp2 2>/dev/null || echo 'failed' || grep 'failed' 
```

Comment:
this will always succeed regardless of the output of the `popper` command.

This can be an alternative:

```bash
set +e
popper init --stages=one,teardown,three sample_exp2
test $? -ne 0
set -e
```